### ISSUE TITLE
Where are the validated images saved?

### ISSUE THREAD

**Issue by VickkyMama** [zhengchen1999/CAT#10]
Hi,

I am trying to train from scratch using my own dataset and I cannot find where the image results are saved. Inside the experiments folder, only the models and training states are saved. The visualisation folder is empty, I can't find the validation images too. HELPPP

**Comment by zhengchen1999** [comment]
Hi.

By default, validation images are not saved (`save_img: False`). If saving is required, you need to set in the [yml](https://github.com/zhengchen1999/CAT/blob/main/options/train/train_CAT_A_sr_x2.yml#L89) file: `save_img: True`.

**Comment by VickkyMama** [comment]
Sorted but it is showing l_pix: nan? and the visualisation is just empty black.

INFO: [train..][epoch:  0, iter:  40,200, lr:(1.000e-04,)] [eta: 4 days, 12:11:27, time (data): 1.469 (0.000)] l_pix: nan

**Comment by zhengchen1999** [comment]
This is strange. It seems like the exploding gradients. I have never encountered this situation. You can try adding `torch.nn.utils.clip_grad_norm_(self.net_g.parameters(), 0.01)` between Line 113 and 114 in [sr_model](https://github.com/zhengchen1999/CAT/blob/main/basicsr/models/sr_model.py#L113) to clip the gradient.

If it still doesn't work, you can check if the input data is normal.
